src/styles/_catalog.scss

```diff
@@ -1,22 +1,40 @@
 .catalog {
```

Comment:
The layout a little bit differ from the one built on the display: flex. 
Probably, you need to add  justify-content: center.